src/org/jitsi/impl/neomedia/rtp/translator/RTPConnectorImpl.java

```diff
@@ -201,6 +209,11 @@ public synchronized PushSourceStream getControlInputStream()
     public synchronized OutputDataStreamImpl getControlOutputStream()
         throws IOException
     {
+        if (this.closed)
+        {
+            return null;
```

Comment:
Well, there were several places in the code where this was already handled that connector can return null and in this case, we skip the writing... it looks safer this way, cause I'm not sure wouldn't IllegalStateException break something else ... 
You are right, but I would insist on merging this and work on the other approach later, because:
1. I want to merge current jitsi, libjitsi and jigasi changes cause we continue seeing losing jigasi instances on meet.jit.si
2. And I continue the work on improving jigasi calling even after these changes, as there are more issues I had found under high load and during that work I can test and work on that.

Thanks for looking at it.